### PR TITLE
chore(common): remove rust (for now!) 🦀

### LINKED ISSUE #6290: chore(common): remove rust (for now!) 🦀
followon to #5069

### PR DESCRIPTION
**Pull request by srl295**
Fixes #6290

### CODE REVIEW

**Comment by srl295** [comment]
@keymanapp-test-bot skip

**Comment by srl295** [comment]
@mcdurdin  does wasm-pack need to go also?

**Review by ermshiperete — APPROVED** [review]
lgtm

**Comment by mcdurdin** [comment]
Windows build failed because keyman32.vcxproj and keyman64.vcxproj both import keyman_keyboard_processor_native.lib:

```
[10:01:01][Step 1/1] 
[10:01:01][Step 1/1] "C:\BuildAgent\work\7ac43416c45637e9\keyman\windows\src\engine\keyman32\keyman32.vcxproj" (Rebuild target) (1) ->
[10:01:01][Step 1/1] (Link target) -> 
[10:01:01][Step 1/1]   LINK : fatal error LNK1181: cannot open input file 'keyman_keyboard_processor_native.lib' [C:\BuildAgent\work\7ac43416c45637e9\keyman\windows\src\engine\keyman32\keyman32.vcxproj]
[10:01:01][Step 1/1] 
[10:01:01][Step 1/1]     0 Warning(s)
[10:01:01][Step 1/1]     1 Error(s)
[10:01:01][Step 1/1] 
[10:01:01][Step 1/1] Time Elapsed 00:00:05.20
[10:01:01][Step 1/1] NMAKE :  U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\\MSBuild\Current\Bin\msbuild.exe"' : return code '0x1'
```

File: common/core/desktop/build.sh [hunk 1]
```diff
@@ -103,66 +91,46 @@ while [[ $# -gt 0 ]] ; do
...
     build)
       HAS_TARGET=true
-      BUILD_RUST=true
       BUILD_CPP=true
       ;;
-    build-rust)
-      HAS_TARGET=true
-      BUILD_RUST=true
+    *-rust)
+      echo "$key: Rust was removed in <https://github.com/keymanapp/keyman/issues/6290>"
```
Comment (mcdurdin) [inline common/core/desktop/build.sh:104]: :+1:

**Review by mcdurdin — COMMENTED** [review]
This will be LGTM once the Windows build passes :smile:. It's both painful and pleasing to see all those rust tendrils stripped out -- so much work to get them going the first time, but so messy, so it's good to see them gone again!

**Review by mcdurdin — APPROVED** [review]
All builds passed, LGTM!

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.203-alpha](https://keyman.com/downloads/releases/alpha/15.0.203)

**Pull request merged**